docker/Dockerfile

```diff
@@ -71,10 +71,7 @@ RUN pip3 install --upgrade setuptools
 RUN pip3 install -r requirements.txt
 RUN python3 setup.py install
 
-WORKDIR /tmp/src/
-COPY start.sh /tmp/src/start.sh
-
-CMD bash /tmp/src/start.sh
+CMD bash /tmp/src/openeo_udf/docker/start.sh
```

Comment:
since we have already cloned the github repository we should overwrite the start.sh with the local one. This caused troubles when using the python udfs on a windows host (carriage returns).